sound/hda/intel-dsp-config.c

```diff
@@ -31,7 +32,12 @@ struct config_entry {
 	u16 device;
 	u8 acpi_hid[ACPI_ID_LEN];
 	const struct dmi_system_id *dmi_table;
-	u8 codec_hid[ACPI_ID_LEN];
+	const struct snd_soc_acpi_codecs *codec_hid;
```

Comment:
@plbossart, out of curiosity, what 'hid' stands for? Can this be simply named as 'codecs'?

Reply:
It's an ACPI acronym for 'Hardware ID'. There's also 'Compatible ID'.
this is represented in DSDT, e.g.

````      
Device (ESSX)
                {
                    Name (_ADR, Zero)  // _ADR: Address
                    Name (_HID, "ESSX8336")  // _HID: Hardware ID
                    Name (_CID, "ESSX8336")  // _CID: Compatible ID
                    Name (_DDN, "ES8336")  // _DDN: DOS Device Name
````